hpcgap/lib/oper1.g

```diff
@@ -462,7 +462,7 @@ BIND_GLOBAL( "INSTALL_METHOD",
 
       # find the operation
       req := GET_OPER_FLAGS(opr);
-      if req = false  then
+      if req = fail  then
```

Comment:
Good catch, this slipped with b1a0e86e43e11bab71d2aef89fb54529f2fe4f24 from PR #1707 by @frankluebeck and I failed to notice it during review :-(